### ISSUE TITLE
Would not add post to blogger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#726]
```
What's the problem?

 I tried to add a post to my blogger site and I added the blogger site to scribefire.  The message said" well this is embarrassing ". When I looked at scribe fire it does not look like it recognized the addition of the blog site.


What browser are you using?

 I just deleted Firefox completely from my machine and then reinstalled it. I had scribe fire working before; however, it does not appear to be working now. I will try again on another page.



What version of ScribeFire are you running?

```

-----
Original issue reported on code.google.com by `carova...@gmail.com` on 21 Jan 2013 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
i am having the same issue...

Jay
```

-----
Original comment by `MichM...@gmail.com` on 1 Feb 2014 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
fixed with latest 4.3.1 version
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:52
* Changed state: **Fixed**
